user_guide_src/source/database/metadata.rst

```diff
@@ -88,7 +86,7 @@ Returns an array of objects containing field information.
 Sometimes it's helpful to gather the field names or other metadata, like
 the column type, max length, etc.
 
-.. note:: Not all databases provide meta-data.
+.. note:: Not all databases provide metadata.
```

Comment:
It would be more clear if it is specified exactly which of the databases provides the Metadata.

Reply:
Does that just come from CI3's documentation?
https://www.codeigniter.com/userguide3/database/metadata.html#retrieve-field-metadata

At least, the current supported 5 databases provide metadata.
See the child classes of `AbstractGetFieldDataTestCase`.

Reply:
I don't know exactly what this note means.
We already have command `php spark db:table --metadata`, we need to check if the test for it is fully written, if the test is complete then this note seems to have expired.

Reply:
We have already the live test cases for the `getFieldData()`.
See the test case files: https://github.com/search?q=repo%3Acodeigniter4%2FCodeIgniter4%20AbstractGetFieldDataTestCase&type=code